src/libs/ajax/SamResources.ts

```diff
@@ -1,18 +1,24 @@
 import _ from 'lodash/fp';
 import { appIdentifier, authOpts, fetchSam, jsonBody } from 'src/libs/ajax/ajax-common';
 
+type RequesterPaysProject = undefined | string;
+
 export const SamResources = (signal: AbortSignal) => ({
   leave: (samResourceType, samResourceId): Promise<void> =>
     fetchSam(
       `api/resources/v2/${samResourceType}/${samResourceId}/leave`,
       _.mergeAll([authOpts(), appIdentifier, { method: 'DELETE' }])
     ),
 
-  getSignedUrl: async (bucket: string, object: string, project: string, requesterPays = false): Promise<string> => {
+  getSignedUrl: async (
+    bucket: string,
+    object: string,
+    requesterPaysProject: RequesterPaysProject = undefined
+  ): Promise<string> => {
     const res = await fetchSam(
-      `api/google/v1/user/petServiceAccount/${project}/signedUrlForBlob`,
+      'api/google/v1/user/signedUrlForBlob',
```

Comment:
Is seems weird to make this use an arbitrary PSA...but I mentioned that in another PR